src/lib/moment/to-type.js

```diff
@@ -29,6 +29,6 @@ export function toObject () {
 }
 
 export function toJSON () {
-    // JSON.stringify(new Date(NaN)) === 'null'
-    return this.isValid() ? this.toISOString() : 'null';
+    // JSON.stringify(new Date(NaN)) === null
```

Comment:
Updated, thanks for the reminder, didn't think that one through enough. Fixed in commit `dafaba3`